### DISCUSSION TITLE
Joining radix efforts

### DISCUSSION THREAD

**Discussion by erlend-sh** [Upbolt/leptos_primitives#7] [General]
There appears to be another radix porting project being done by @DanielleHuisman here: https://github.com/NixySoftware/radix

Maybe your two projects could be merged together? ☺️

**Comment by Upbolt** [comment]
i'll see if i can get in contact

**Comment by DanielleHuisman** [comment]
I'd be open to collaborating. I'm going to look at your project and get back to you. It might take a few days before I have time.

**Reply by Upbolt** [reply]
sounds good, also do you happen to have a discord account so we can have a means of more immediate contact?

**Comment by R2rho** [comment]
Just wanted to say I'm super hyped for this project! If I can join the discord I would like to be a part of this!

**Reply by Upbolt** [reply]
currently i've just been discussing things in the leptos discord server, feel free to join and share opinions 😁

**Reply by DanielleHuisman** [reply]
I've joined the Leptos Discord. Feel free to tag me on there if you continue discussing this project.
